deepmd/pt/model/model/dp_zbl_model.py

```diff
@@ -24,7 +24,7 @@
 
 
 @BaseModel.register("zbl")
-class DPZBLModel(DPZBLModel_):
+class DPZBLModel(DPModelCommon, DPZBLModel_):
```

Comment:
DPZBLModel should not be derived from `DPModelComm`, because it is not a DP model.